data-otservbr-global/npc/gnomargery.lua

```diff
@@ -148,15 +148,15 @@ local function creatureSayCallback(npc, creature, type, message)
 	////PARCEL DELIVERY////
```

Comment:
These types of comments can also be removed, they don't help at all and make the code ugly.